Corona-Warn-App/src/main/java/de/rki/coronawarnapp/eventregistration/checkins/TraceLocationCheckInRepository.kt

```diff
@@ -0,0 +1,28 @@
+package de.rki.coronawarnapp.eventregistration.checkins
+
+import de.rki.coronawarnapp.eventregistration.storage.TraceLocationDatabase
+import de.rki.coronawarnapp.eventregistration.storage.dao.TraceLocationCheckInDao
+import de.rki.coronawarnapp.eventregistration.storage.entity.TraceLocationCheckInEntity
+import kotlinx.coroutines.flow.Flow
+import javax.inject.Inject
+
+class TraceLocationCheckInRepository @Inject constructor(
+    traceLocationDatabaseFactory: TraceLocationDatabase.Factory
```

Comment:
To follow the same pattern with TraceLocationRepository. All operations in the repos should be launched in `AppScope` see https://github.com/corona-warn-app/cwa-app-android/pull/2549#discussion_r589519873

Reply:
inserting in appscope, no change on reading data, because that's done in a flow